dyn_em/module_first_rk_step_part1.F

```diff
@@ -579,6 +581,7 @@ SUBROUTINE first_rk_step_part1 (   grid , config_flags              &
      &        ,LakeModel=grid%sf_lake_physics,  lake_min_elev=grid%lake_min_elev    & !lake  
 #if ( EM_CORE == 1)
      &        ,LakeMask=grid%LakeMask                                               & ! lake
+     &        ,restart_flag=restart_flag                                               & ! is it a restart timestep?
```

Comment:
@negin513 
Negin,
Line up the trailing "&" + comment indenting/spacing. This is probably easiest from a sandbox, as I can never guess space vs tab, etc

Reply:
Thanks for your comment. I've addressed your comment in the recent push.